Printer.java

```diff
@@ -36,6 +48,81 @@ public void info(String s) {
     }
 
     public void error(String s) {
-        err.println(new AttributedString(s, AttributedStyle.DEFAULT.foreground(AttributedStyle.RED)).toAnsi());
+        err.println(colorError(s));
+    }
+
+    public void conceptMap(ConceptMap conceptMap, Grakn.Transaction tx) {
+        out.println(getConceptMapDisplayString(conceptMap, tx));
+    }
+
+    private String getConceptMapDisplayString(ConceptMap conceptMap, Grakn.Transaction tx) {
+        StringBuilder sb = new StringBuilder();
+        sb.append("{ ");
+        for (Map.Entry<String, Concept> entry: conceptMap.map().entrySet()) {
+            String variable = entry.getKey();
+            Concept concept = entry.getValue();
+            sb.append(variable);
+            sb.append(" ");
+            sb.append(getConceptDisplayString(concept, tx));
+            sb.append("; ");
+        }
+        sb.append("}");
+        return sb.toString();
+    }
+
+    private String getConceptDisplayString(Concept concept, Grakn.Transaction tx) {
```

Comment:
This code is largely translated over from the old console, we didn't really reason about if that's the correct thing to display.

Reply:
can we extract the code inside the branches onto its own function, and use this naming convention:

```
conceptDisplayString():

if (concept instanceof Attribute<?>) {
  attributeDisplayString(...)
} else if (concept instanceof Type) {
  typeDisplayString(...)
}
...
```